README.md

```diff
@@ -14,61 +14,148 @@ At Pensano Developers, we are committed to giving back to our community and supp
 We work in agile environments, test-driven development, object-oriented programming, and quickly adapting to new tech stacks. We aim to deliver software that meets the needs of the customer, is flexible and adaptable to change, and is delivered efficiently and with high quality.
 
 # Getting started
+
 1. Clone the repo
 2. Install Node.js dependencies for both front and backend directories
 
-  ```bash
-  cd backend
-  npm install
-  cd ../frontend
-  npm install
-  ```
-3. Install the "react-allergens" package using the package manager of your choice, such as npm or yarn.
-
-To install "react-allergens" using npm:
-  ```bash
-  npm install react-allergens
-  ```
+```bash
+cd backend
+npm install
+cd ../frontend
+npm install
+```
+
+3. Ensure nodemon is installed by running
+
+```bash
+npm install nodemon --global
+```
+
+4. Install the "react-allergens" package using the package manager of your choice, such as npm or yarn.
```

Comment:
Similar to nodemon, but why do we need this when it is already a dependency? Is there any known issue here? If there is then we should be expected to have a production issue with it.